README.rst

```diff
@@ -46,8 +46,14 @@ How do I learn more?
   detail the motivation for Charliecloud and the technology upon which it is
   based: https://www.usenix.org/publications/login/fall2017/priedhorsky
 
-* A more technical resource is our Supercomputing 2017 paper:
-  https://dl.acm.org/citation.cfm?id=3126925
+* A more technical resource is our Supercomputing 2017 paper: https://doi.org/10.1145/3126908.3126925
+
+* Our approach to deploying scientific applications in HPC using Charliecloud’s layer-free, Git-based container 
+  build cache: https://doi.org/10.1145/3624062.3624585
+
+* A performance comparison between three HPC-specific container technologies against a bare metal implementation: https://ieeexplore.ieee.org/document/8950978
+
+* Demonstration of how low-privilege containers solve the increasing demand of software flexibility: https://doi.org/10.6084/m9.figshare.14396099
```

Comment:
Listing the links twice seems awkward to me. One possible solution: remove the technical papers from here, change "how do I credit" to "Publications", and add a pointer to that section below.

That said, I really like how you added a 1-sentence summary for each paper and I want to keep those. Let’s re-frame those into a description of the findings/outcome rather than what we did, though.